### PR TITLE
Add hugme script

### PR DESCRIPTION
**Pull request by dlapiduz**

### CODE REVIEW

File: scripts/hugme.coffee [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#   None
+#
+# Commands:
+#   hubot hug me - Receive a hug
+#   hubot hug bomb N - get N hugs
+
+module.exports = (robot) ->
+
+  hug_count = parseInt(process.env.HUG_COUNT)
+  base_url = process.env.HUG_BASE_URL
```
Comment (afeld) [inline scripts/hugme.coffee]: Where are you thinking of storing them?

Comment (dlapiduz) [inline scripts/hugme.coffee]: S3... the base url is just an S3 bucket and it is 1.jpg, 2.jpg etc

Comment (afeld) [inline scripts/hugme.coffee]: Maybe add a note about that in the comment at the top? Otherwise :shipit:.

**Comment by meiqimichelle** [comment]
yessssss yes yes yes

File: scripts/hugme.coffee [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   hubot hug me - Receive a hug
+#   hubot hug bomb N - get N hugs
+#
+# Notes:
```
Comment (dlapiduz) [inline scripts/hugme.coffee]: @afeld does something like this work?

**Pull request merged**